libraries/ESP8266WiFi/src/ESP8266WiFiSTA.h

```diff
@@ -49,6 +49,7 @@ class ESP8266WiFiSTAClass: public LwipIntf {
 
         // two and one parameter version. 2nd parameter is DNS like in Arduino
         // IPv4 only
+        [[deprecated("It is discouraged to use this 1 or 2 parameters network configuration legacy function config(ip[,dns]) because it is insufficiently accurate")]]

```

Comment:
"insufficiently accurate" is a bit confusing.
Maybe ".. as chosen defaults may not be universally applicable"
Or ".. as chosen defaults may not match the local network configuration"

I think the last one may be most telling to a user that his program might work at his desk, but not for the neighbor he's making the device for.

Reply:
done